### DISCUSSION TITLE
Improve Update available and Check for updates

### DISCUSSION THREAD

**Discussion by jsoref** [Kong/insomnia#3675] [Ideas]
**Is your feature request related to a problem? Please describe.**
I had Insomnia open, and heard elsewhere that there was a "new version". I went and searched the menus for an updates item and didn't find one. I went to the about dialog and saw that I had an old version, but didn't find any way to update.

**Describe the solution you'd like**
* Users should be able to visually see there's an update available, w/o staring at their screens 24x7 (i.e. growl notifications are not the way to tell users about updates)
* Users should be able to check for updates by either using help> or app> and either seeing a menu item for "check for updates" or seeing a button in the about dialog

**Describe alternatives you've considered**
The fact that it's possible through preferences is true, but it is not the expected results of users today.

**Additional context**
<details><summary>Pictures of various update flows</summary>

<details><summary>Chrome</summary>

<details><summary>Chrome update indicator</summary>

![chrome caution version somewhat stale](https://user-images.githubusercontent.com/2119212/116729317-056aeb80-a9b5-11eb-989f-bb5359baa114.png)
![chrome new update available](https://user-images.githubusercontent.com/2119212/116729333-0a2f9f80-a9b5-11eb-8d88-062b6f8911f7.png)
![chrome version dangerously insecure](https://user-images.githubusercontent.com/2119212/116729345-0d2a9000-a9b5-11eb-9ee0-1f52005fab64.png)
</details>

<details><summary>Chrome about</summary>

![Chrome about checking for updates](https://user-images.githubusercontent.com/2119212/116729490-39461100-a9b5-11eb-8587-11d9072912f3.png)
</details>

</details>
<details><summary>Firefox</summary>

<details><summary>Firefox update indicator</summary>

![Firefox update indicator](https://user-images.githubusercontent.com/2119212/116729360-10be1700-a9b5-11eb-865c-59b4f6a53bf7.png)

</details>

<details><summary>Firefox about</summary>

![firefox about checking for updates](https://user-images.githubusercontent.com/2119212/116729536-46fb9680-a9b5-11eb-836f-53d1daaf3255.png)
![about install update](https://user-images.githubusercontent.com/2119212/116729355-0ef45380-a9b5-11eb-9fe7-ef7baf00e805.png)

</details>

</details>

<details><summary>Check for updates in menus</summary>

![slack](https://user-images.githubusercontent.com/2119212/116729415-203d6000-a9b5-11eb-98e8-e7dfda7e1b17.png)
![outlook](https://user-images.githubusercontent.com/2119212/116729454-29c6c800-a9b5-11eb-9360-626000f726af.png)

</details>

**Comment by DanielRouxSA** [comment]
I know this discussion is over a year old now without any activity, but I'd very much like to see this moved forward; the current update workflow for Insomnia is rather inelegant compared to almost every other app I've used that incorporates self-updating.
